content/release-notes/2022-11-17.md

```diff
@@ -0,0 +1,27 @@
+---
+label: 'Console'
+---
+
+### What's new
+
+- UI: Postgres 15 is now the default version when creating a new project.
+- UI: Branches can be renamed in the single branch view screen.
+- UI: Added timestamp and lsn to the branch view **TODO**: find out what it is https://github.com/neondatabase/cloud/pull/2919/commits/d05c714845e4550fa7f460755e9bbb8c778e00fe
```

Comment:
@seymourisdead can you please clarify "add timestamp and lsn to the branch view" for [this commit](https://github.com/neondatabase/cloud/pull/2919/commits/d05c714845e4550fa7f460755e9bbb8c778e00fe)?